cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -12,116 +12,184 @@ Replacement syntax for deprecated and removed features are also indicated.
 
 [[cypher-deprecations-additions-removals-4.3]]
 == Version 4.3
+
+=== Deprecated features
```

Comment:
Hmm, I haven't changed the order. You can find the original on line 238.
It looks like this:

a|
label:syntax[]
label:deprecated[]
[source, cypher, role="noheader"]
----
SHOW EXISTS CONSTRAINTS
----
a|
Replaced by:
[source, cypher, role="noheader"]
----
SHOW [PROPERTY] EXIST[ENCE] CONSTRAINTS
----
Still allows `BRIEF` and `VERBOSE` but not `YIELD` or `WHERE`.

Reply:
Not what I ment, lets see if I can explain my thoughts better

You ordered the categories as:
```
deprecated
updated
new
```
while when we (or at least me, but I think more of us cypher people) added things to this list we would add them in the order
```
new or updated
deprecated
```
so we first introduced the replacement and then the deprecation. The description for the deprecation would still include the replacement which is what your piece of code is.

another example would be first introducing dropping index by name (`DROP INDEX name`) and then deprecating dropping index by label/property combination (`DROP INDEX ON :Label(prop)`).

Is that any clearer?

Reply:
Yes, now I understood you :) OK, so I changed the order because the requested improvement was to add some kind of structure, especially to the areas that are affected, such as removals and deprecations, because those will break code going forward while additions are mostly FYI.